Maintain compatibility with Python 3.13

daskol opened the pull request:
Resolve #371.

daskol commented:
Wheels `dm-tree` for Python 3.13 are missing. Building `dm-tree` from tarball with default `gcc` fails because of missing `typedef`s of `uint8_t` and `uint64_t` in `absl/strings/internal/str_format/extension.h` of vendored `abseil` library. Upstream issue is https://github.com/google-deepmind/tree/issues/119.

medaminezghal commented:
@daskol try to remove the "python_version < "3.13"" in the "https://github.com/google-deepmind/chex/blob/master/requirements/requirements-test.txt" file and rebuild the package.

daskol commented:
@jakevdp Take a look at this PR. It became stale until I added support for Python 3.13 in https://github.com/google-deepmind/tree/pull/123. Then Sergei Lebedev uploaded cp313 wheels on the last days of January.

This branch is squashed and rebased on top of the latest changes in `master`.

medaminezghal commented:
@stompchicken Could you approve this change and update the package to be compatible with python 3.13?
